### PR TITLE
Add screen width detection for collapsible sections on articles

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

**Comment by nlisgo** [comment]
@discodavey, I have added some comments to the relevant issue: https://github.com/elifesciences/issues/issues/2900#issuecomment-740137215

**Comment by nlisgo** [comment]
@discodavey I am happy with this now. Please take a look.

File: test/articlesection.spec.js [hunk 1]
```diff
@@ -177,13 +190,4 @@ describe('An Article Section (collapsible)', function () {
     expect(section.$body.classList.contains('visuallyhidden')).to.be.false;
   });
 
-  describe('obtaining the hash', function () {
```
[the remark this answers is not in the document]
Comment (discodavey) [reply test/articlesection.spec.js:180]: Looks good to me.

[the remark this answers is not in the document]
Comment (discodavey) [reply test/articlesection.spec.js:180]: @nlisgo Anything else I need to check?

**Review by nlisgo — APPROVED** [review]

**Pull request merged**